src/bin/pants/cli.rs

```diff
@@ -57,5 +57,9 @@ pub enum Opt {
         /// OSS Index API Key
         #[structopt(long = "ossi-api-key", env, hide_env_values = true)]
         oss_index_api_key: Option<String>,
+
+        // Ignore file
+        #[structopt(short = "i", long = "ignore-file")]
+        ignore_file: Option<String>,
```

Comment:
We might want this to be `PathBuf` to get some filepath validity handling for free

Reply:
In: 498e0a1